src/zino/tasks/reachabletask.py

```diff
@@ -20,17 +21,17 @@ def __init__(self, *args, **kwargs):
     async def run(self):
         """Checks if device is reachable. Schedules extra reachability checks if not."""
         if self._extra_job_is_running():
-            return
+            raise DeviceUnreachableError
```

Comment:
Might be worth mentioning this in a comment before that line.

Reply:
> Of course, it could possibly be argued that if this job is run again before the scheduled "extra job", then the "extra job" is redundant - but this is the current logic of reachability testing...

The point of the extra job is that it runs more often than the regular job. If the box is down for a long time, then they will overlap, and this case happens.